modules/authentication/src/admin/admin.ts

```diff
@@ -0,0 +1,405 @@
+import ConduitGrpcSdk, {
```

Comment:
This file is a rebase leftover. Move to `modules/authentication/src/admin/index.ts`